.circleci/config.yml

```diff
@@ -5,46 +5,38 @@ shared_configs:
         name: Run tests
         command: |
           make deps test
-
+  full_job_steps: &full_job_steps
+    - checkout
+    - run:
+        name: Run tests
+        command: |
+          make ci
 
 # Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
 version: 2.1
 jobs:
   build-1-15:
     working_directory: ~/repo
     docker:
-      - image: circleci/golang:1.15
+      - image: cimg/go:1.15
     steps: *simple_job_steps
 
   build-1-16:
     working_directory: ~/repo
     docker:
-      - image: circleci/golang:1.16
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - go-mod-v4-{{ checksum "go.sum" }}
-      - run:
-          name: Install Dependencies
-          command: go mod download
```

Comment:
We don't need to run this command anymore?

Reply:
Nah, the necessary modules will be downloaded when it tries to build.  This was broken out before in order to try to populate Go's module cache and then persist it to Circle's cache.  I found this to be a net loss; restoring the cache could take several minutes, while the non-cached versions would completely in like 1 minute.